playbooks/hammer-devel.yml

```diff
@@ -0,0 +1,4 @@
+---
+- hosts: hammer-devel
```

Comment:
If you add 'all' here, it would allow me to do this against my own box however I want, e.g.

```
ansible-playbook -l myhost playbooks/hammer-devel.yml